### PR TITLE
Change to src layout and release 1.0.3

### PR DESCRIPTION
**Pull request by naruxde**
Reorder project to src layout and add some pythonize files for build and deploy process.

Makefile can create a virtual environment
Makefile can build source and wheel distribution
Makefile can install package to test it in the venv

Version number for entire project in about.py only (setup.py uses this one).

Parameter --version always outputs the version of the software on the CLI.

Long description is generated from README.md (also for PyPi)

### CODE REVIEW

**Comment by iluminat23** [comment]
SPDX tags are missing. Add them or create a ticket, that this needs to be done!

**Comment by nbuchwitz** [comment]
@iluminat23 @naruxde I've added the SPDX headers and pushed to the PR. The rest LGTM

**Comment by naruxde** [comment]
You both have my teams contact, we are working in the same company, why chatting here and not talk directly about this?

**Review by nbuchwitz — APPROVED** [review]

**Review by iluminat23 — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,48 @@
-from setuptools import setup, find_packages
+# -*- coding: utf-8 -*-
```
Comment (iluminat23) [inline setup.py:1]: Missing SPDX tag

Comment (naruxde) [reply setup.py:1]: setup.py is a default file, like Makefile, manifest.in and so on, do that files need the SPDX as well?

Comment (iluminat23) [reply setup.py:1]: Is this file generated?

Comment (naruxde) [reply setup.py:1]: As much as Makefile, manifest.in and so on...

Comment (iluminat23) [reply setup.py:1]: The only files which clearly don't need a license header are generated files. All other files should contain a license header. It is not a big issue, but I was just a minute to late to realize the issue. I had written the comment but didn't hit the button until you merged the branch. So ...

Comment (naruxde) [reply setup.py:1]: Link the law, don't see the need in that kind of files, you can autogenerate all of that... So no SPDX needed...

Comment (naruxde) [reply setup.py:1]: BUT: What about Markdown files? Do we have to add a SPDX commend? @iluminat23

**Comment by iluminat23** [comment]
Hmmm, this is a good question. A GPL or MIT would not do any good with these kind of files. And I have no idea what license our documentation should have.

File: setup.py (hunk 1 above)
Comment (iluminat23) [reply setup.py:1]: Hmmm, this is a good question. A GPL or MIT would not do any good with these kind of files. And I have no idea what license our documentation should have.

**Pull request merged**